### PR TITLE
wgengine/wgcfg/nmcfg: consolidate "skipped unselected" log lines

### PR DESCRIPTION
**Pull request by josharian**
These were getting rate-limited for nodes with many peers.
Buffer the output and emit a single line, which is nicer anyway.

### CODE REVIEW

File: wgengine/wgcfg/nmcfg/nmcfg.go [hunk 1]
```diff
@@ -60,6 +61,7 @@ func WGCfg(nm *netmap.NetworkMap, logf logger.Logf, flags netmap.WGConfigFlags,
 		Peers:      make([]wgcfg.Peer, 0, len(nm.Peers)),
 	}
 
+	skippedUnselected := new(bytes.Buffer)
```
Comment (bradfitz) [inline wgengine/wgcfg/nmcfg/nmcfg.go:64]: what about the line 108 ones?

Comment (josharian) [reply wgengine/wgcfg/nmcfg/nmcfg.go:64]: They aren't getting rate limited, so I didn't bother with them. If you'd like I can give them all the same treatment.

Comment (josharian) [reply wgengine/wgcfg/nmcfg/nmcfg.go:65]: Did the rest.

**Review by bradfitz — APPROVED** [review]
Little sad about the garbage, but it's not obviously a win to use logger.ArgWriter here so I guess this is okay.

**Pull request merged**